src/error/syntaxError.js

```diff
@@ -22,9 +22,11 @@ export function syntaxError(
   description: string
 ): GraphQLError {
   const location = getLocation(source, position);
+  const line = location.line + source.location.line - 1;
```

Comment:
I suspect we don't want to name this `source.location`, to avoid confusion with the existing `SourceLocation` type and `getLocation` function. Maybe we could call it something like `base` or `offset` or something else.